medusa/history.py

```diff
@@ -22,7 +22,7 @@
 import db
 from .common import FAILED, Quality, SNATCHED, SUBTITLED
 from .helper.encoding import ss
-from .show.History import History
+from .show.history import History
```

Comment:
`from .show.history` already differentiates between `show/history.py` and `history.py`.  If `history.py` also had a `class History:` then we could import it as ShowHistory to differentiate, but thats not the case here.